docker/scripts/entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+source /root/.bash_profile; \
+/bin/bash /cbio_scripts/addcBioAliases.sh; \
+/bin/bash /cbio_scripts/checkCustomConfig.sh; \
+$CATALINA_HOME/bin/startup.sh; \
```

Comment:
I would suggest `CMD` instead, to allow for running different commands (such as the data loading script) instead of the web server. `CMD` sets the default command if no other command is given to `docker run`, while `ENTRYPOINT` seems to be intended for sh scripts that configure the container (the part that varies between installations) before they exec the command. For instance, the Dockerfile for MySQL uses it to set up the database and user permissions before it execs `mysqld`: https://github.com/docker-library/mysql/tree/master/5.7